src/capidle/script/util.js

```diff
@@ -0,0 +1,6 @@
+// this file contains helper functions and constants
```

Comment:
What is the convention for when to use multi-line and single-line JavaScript comments? (There's also an extra asterisks in here btw)

Reply:
I'm not sure what the convention is, but using `/** hello  */` (double star) enables [JSDoc](https://jsdoc.app/about-getting-started.html)/[JavaDoc](https://www.baeldung.com/javadoc) syntax in many languages. So a double-star comment will show up when you hover over that function anywhere in the code. 

<img width="199" alt="image" src="https://user-images.githubusercontent.com/16009897/183586822-f3b49b7c-cda0-49c8-bb25-445983f45ec6.png">